src/components/table/column-picker/column-picker.component.ts

```diff
@@ -41,60 +43,174 @@ export class ColumnPickerComponent {
     /** Define a function that return a column move announcement */
     @Input() columnMovedAnnouncement: (column: string, delta: number) => string = this.getColumnMovedAnnouncement;
 
+    /** group settings for customising display */
```

Comment:
We should aim to reuse the text from the documentation for doc comments, it will help if we ever get round to generating docs from source.